### DISCUSSION TITLE
use or ignore multichannel exr channels

### DISCUSSION THREAD

**Discussion by flinfo** [ImageMagick/ImageMagick#6641] [Development] [answered]
### ImageMagick version

7.0.10-30

### Operating system, version and so on

centos 7.9

### Description

From my understanding, multichannel exr's are not supported yet.

error:
```
convert mmmm.xxxx.exr mmmm.xxxx.jpg
convert: unable to open image 'Cannot read image file "mmmm.xxxx.exr". The file format version number's flag field contains unrecognized flags.': Success @ error/exr.c/ReadEXRImage/202.
```

The 'identify' command results in the exact same error. Imagemagick just chokes dead on the multiple channels, so cannot get to the point that any information about the file is available to display.
But is there a way I can ignore the multichannels and just use the standard rgb (I am assuming there is a standard RGB in multichannel exrs)

TIA for any thoughts

**Comment by urban-warrior** [comment, marked as answer]
The exception is thrown by the OpenEXR delegate library, specifically the ImfOpenInputFile() API call.  If you are using an older release of OpenEXR, upgrade and see if that resolves the issue (we're using version 3.1.1), otherwise contact the OpenEXR developers and see if they can provide insight or a patch for the issue.

**Reply by flinfo** [reply]
Thanks for the clear answer, much appreciated!!
